components/infobox/wikis/worldofwarcraft/infobox_team_custom.lua

```diff
@@ -0,0 +1,50 @@
+---
+-- @Liquipedia
+-- wiki=worldofwarcraft
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local REGION_REMAPPINGS = {
+	['cis'] = 'europe',
```

Comment:
Can't find any usages of CIS on WoW

Might be the case of the other regions too?